handbook/demand/README.md

```diff
@@ -280,6 +280,16 @@ Here are some links that filter relevant conversations on each platform:
 4. If a post is helpful to our audience, reshare it.
 
 
+### Record and edit Sprint demos
+Every three weeks there's a sprint demos meeting on the office hours calendar. Demand is responsible for utilizing the Gong recording and preparing it for YouTube and other social media.
+
+1. Download the most recent Sprint demos video from [Gong](https://us-65885.app.gong.io/conversations?workspace-id=9148397688380544352&callSearch=%7B%22search%22%3A%7B%22type%22%3A%22And%22%2C%22filters%22%3A%5B%7B%22type%22%3A%22CallTitle%22%2C%22phrase%22%3A%22sprint%20demos%22%7D%5D%7D%7D)
```

Comment:
```suggestion
1. Download the most recent sprint demos video from [Gong](https://us-65885.app.gong.io/conversations?workspace-id=9148397688380544352&callSearch=%7B%22search%22%3A%7B%22type%22%3A%22And%22%2C%22filters%22%3A%5B%7B%22type%22%3A%22CallTitle%22%2C%22phrase%22%3A%22sprint%20demos%22%7D%5D%7D%7D).
```

Sentence case: https://fleetdm.com/handbook/company/communications#sentence-case 

"Sprint demos" isn't a proper now in this case.